client/src/components/ServicesFilter/ServicesFilter.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import {listItemSectionBuilder, MultiSelectCheckbox, TableToolbar} from 'wix-style-react';
+import {translate} from '../../utils';
+import {isEqual} from 'lodash';
+
+export default class ServicesFilter extends React.Component {
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            selectedServices: props.filters.services
+        };
+    }
+
+    _getOptionsList = (options) => {
+        const {servicesGroups} = this.props;
+        for (const serviceGroup in servicesGroups) {
```

Comment:
let's try to change this loop to a [reduce](https://thecodebarbarian.com/javascript-reduce-in-5-examples.html) function.